docs/pywbemcli/commands.rst

```diff
@@ -1556,6 +1560,57 @@ Example:
 
     $ pywbemcli --name myserver server remove-mof mymodel.mof
 
+See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
+command.
+
+.. _`Server schema command`:
+
+``server schema`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``server schema`` command returns information on the :ref:`DMTF Schema`
+defined for classes in the namespace()s) of the WBEM server of the
+:term:`current connection`.  It provides information on the maximum value class
+version qualifier (which may represent the DMTF ) in the namespace, whether the
+namespace has experimental classes, the number of classes in the schema, and
+the set of schema prefixes.
+
+pywbemcli cannot determine the actual DMTF schema version of classes that was
+installed in each namespace because the version of the DMTF schema installed is
+not recorded in the installed model so the best that can be done is to find
+the highest value of the Version qualifier that represents the DMTF Schema
+version where the last change was made to that each class.
+
+Example:
+
+.. code-block:: text
+
+    $ pywbemcli --name myserver server schema
+
+    Schema information;
+    Namespace    schemas          classes  schema     experimental
+                                           version
+    -----------  -------------  ---------  ---------  --------------
+    root/cimv2   BLA, EXP, TST         20  2.51.0     Experimental
+
+or a more detailed report (``--detail`` option):
+
+.. code-block:: text
+
+    $ pywbemcli --name myserver server schema --detail
+
+Schema information, detail;
+Namespace    schemas      classes  schema     experimental
+                                   version
+-----------  ---------  ---------  ---------  --------------
+root/cimv2                     20  2.51.0     Experimental
```

Comment:
The schemas column i the row with the totals is not consistent with the code, which says "(namespace-info)" there instead of blanks. However, see my comment on that, further down.

Reply:
Changed per comment below.  Just leaving it, blank is also confusing.